--- a/vendor/thans/tp-jwt-auth/src/provider/JWT/Lcobucci.php
+++ b/vendor/thans/tp-jwt-auth/src/provider/JWT/Lcobucci.php
@@ -23,6 +23,7 @@ use thans\jwt\exception\TokenInvalidException;
 
 class Lcobucci extends Provider
 {
+    protected $signer;
     protected $signers
         = [
             'HS256' => HS256::class,
--- a/vendor/thans/tp-jwt-auth/src/claim/Factory.php
+++ b/vendor/thans/tp-jwt-auth/src/claim/Factory.php
@@ -7,6 +7,7 @@ use think\Request;
 
 class Factory
 {
+    protected $request;
     protected $classMap
         = [
             'aud' => Audience::class,
